src/display/node_utils.js

```diff
@@ -35,19 +35,19 @@ if (isNodeJS) {
 
   const loadPackages = async () => {
     // Native packages.
-    const fs = await __non_webpack_import__("fs"),
-      http = await __non_webpack_import__("http"),
-      https = await __non_webpack_import__("https"),
-      url = await __non_webpack_import__("url");
+    const fs = await __raw_import__("fs"),
+      http = await __raw_import__("http"),
+      https = await __raw_import__("https"),
+      url = await __raw_import__("url");
 
     // Optional, third-party, packages.
     let canvas, path2d;
     if (typeof PDFJSDev !== "undefined" && !PDFJSDev.test("SKIP_BABEL")) {
       try {
-        canvas = await __non_webpack_import__("canvas");
+        canvas = await __raw_import__("canvas");
       } catch {}
       try {
-        path2d = await __non_webpack_import__("path2d");
+        path2d = await __raw_import__("path2d");
       } catch {}
     }
```

Comment:
When built, this is tree-shaken away. This means a consumer of `pdfjs-dist` can't import canvas or path2d. Is this intentional?

(I use the canvas renderer in node, so the latest working version is `4.2.67`)

This is what is [published to npm](https://unpkg.com/browse/pdfjs-dist@4.4.168/build/pdf.mjs).

```ts
  const loadPackages = async () => {
    const fs = await import( /*webpackIgnore: true*/"fs"),
      http = await import( /*webpackIgnore: true*/"http"),
      https = await import( /*webpackIgnore: true*/"https"),
      url = await import( /*webpackIgnore: true*/"url");
    let canvas, path2d;
    return new Map(Object.entries({
      fs,
      http,
      https,
      url,
      canvas,
      path2d
    }));
  };
  ```

Reply:
Well, I missed the fact that the legacy version was required for node in the FAQ. The legacy version retains the ability to import `canvas` and `path2d`.

It turns out the un-suffixed version runs fine in Node 22, sans this canvas issue!